robots/coverage/diff/diff.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package diff calculates the difference of two coverage lists and produces a collection of
+// individual coverage difference. The result is formatted optionally to a table in coverage bot post
+package diff
+
+import (
+	"k8s.io/test-infra/gopherage/pkg/cov/junit/calculation"
+)
+
+const deltaSensitivity = 0.001
```

Comment:
Please leave a code comment justifying this number (like you did in a previous github comment)